packages/pipeline-editor/src/PipelineService.tsx

```diff
@@ -124,4 +127,51 @@ export class PipelineService {
       buttons: [Dialog.okButton()]
     });
   }
+
+  static convertPipeline(pipelineDefinition: any): any {
```

Comment:
why are these functions in the service file and not the util file? They have nothing to do with the elyra api

Reply:
Services in this case are not tied to an API, it's more like a place where "operations" that work on a given "thing" (in this case pipeline) are centralized. Also, I don't like what I did with Utils either :) We need a "wrapper" to manipulate "pipelines" on the front end, but this will probably come later.